### ISSUE TITLE
md2po remove-not-found argument doesn't seem to work

### ISSUE THREAD

**Issue by keunes** [mondeja/mdpo#202]
Given
- the test project https://github.com/keunes/AP-website-copy, with the 'translation' branch containing PO files in `_i18n/PO-files`
- md2po version 0.3.73
- `md2po _i18n/en/general/*.md --po-filepath _i18n/PO-files/site-general_en.PO --save --quiet --remove-not-found`

the [argument to remove not-used messages](https://mdpo.readthedocs.io/en/master/cli.html#md2po---remove-not-found) doesn't seem to have any effect: when I change a text in an MD file, then execute the above code, the message containing the old text is commented out (marked obsolete), but still in the PO file.

**Comment by mondeja** [comment]
This patch has been included in [v0.3.80](https://github.com/mondeja/mdpo/releases/tag/v0.80). Cheers!

**Comment by keunes** [comment]
Nice! Thanks!

**Comment by keunes** [comment]
Hi @mondeja 
I just updated mdpo and have now v0.3.80. It seems to have improved, but not all unused messages are removed: only the last one in the PO file is now removed, so I need to run md2po multiple times in order to remove all obsolete messages from the PO output.

**Comment by keunes** [comment]
Hmmm, weird. After deleting the PO file and let md2po create it from scratch, all unused strings are now removed properly after edits.

**Comment by mondeja** [comment]
I can't reproduce it. Could you share a reproducible example?

**Comment by keunes** [comment]
What do you mean with a 'reproducable example'? My original post already lists the environment and project & files with which I had the original issue. The removing of one obsolete message per time happened like this:
* in v0.3.73, create PO file from MD files
* change some MD files, and update the PO file using the code in my first post (and get a PO with obsolete messages at the end of the file)
* update mdpo to v0.3.80
* execute the command again, and see an (I think one) obsolete message at a time

**Comment by mondeja** [comment]
I've tried it and I can't reproduce it. Could you reproduce it using some other environment? Maybe a repository with only the affected files? Maybe a script? It would be much more easy to debug by this way.

**Comment by keunes** [comment]
Is it possible to downgrade mdpo? If so I'm happy to test if I can reproduce :)

**Comment by mondeja** [comment]
What do you mean with "downgrade mdpo"? You can install whatever version you want with `pip install mdpo==X.Y.Z`

**Comment by keunes** [comment]
@mondeja Ok, see repo https://github.com/keunes/mdpo-bug-test
The PO file contains a few obsolete strings, created by mdpo v 0.3.73.

Running `md2po en/*.md --po-filepath PO-files/site-general_en.PO --save --quiet --remove-not-found` deletes the following two (out of the four) obsolete strings:
```
#~ msgid ""
#~ "You can download official AntennaPod builds on Google Play and on F-Droid."
#~ msgstr ""

#~ msgid ""
#~ "AntennaPod is a podcast player, completely free and respectful of your "
#~ "privacy. It's created by volunteers throughout the world without any "
#~ "commercial interest. Together we contribute to an open podcasting ecosystem."
#~ " So that you get to enjoy the web's best audio content."
#~ msgstr ""
```

These obsolete strings are still there:
```
#~ msgid ""
#~ "Press the button above to subscribe to this podcast in AntennaPod. If you "
#~ "don't have AntennaPod installed yet, you can [download the app "
#~ "here](/download)."
#~ msgstr ""

#~ msgid "Website theme"
#~ msgstr ""
```

Running the same command again removes the `Press the button above...` string (and `Website theme` is left).
Running the same command a third time removes also `Website theme`.

**Comment by mondeja** [comment]
Thank you for the detailed report. It's fixed and released in [v0.3.81](https://github.com/mondeja/mdpo/releases/tag/v0.3.81). Let me know if you experiment any problem 👍🏼
